shared/oae/css/oae.components.css

```diff
@@ -972,9 +972,9 @@ div.oae-thumbnail i.fa {
  *      </a>
  *      <div class="oae-tile-metadata">
  *          <i class="fa fa-oae-public pull-left"></i>
- *          <h3 class="oae-threedots">
+ *          <div class="oae-tile-title oae-threedots">
```

Comment:
Targeting naked elements with style rules tends to make me nervous. Too easy (at least for me) to mess things up later.